packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-accessors--to-es2015/private/output.js

```diff
@@ -24,11 +24,11 @@ function _get_b2() {
   return _get_b(this);
 }
 [_init_a, _get_a, _set_a, _init_b, _get_b, _set_b, _initProto] = babelHelpers.applyDecs(_Foo, [[dec, 1, "a", function () {
-  return babelHelpers.classPrivateFieldGet2(this, _A);
+  return babelHelpers.classPrivateFieldGet2(_A, this);
 }, function (value) {
-  babelHelpers.classPrivateFieldSet2(this, _A, value);
+  babelHelpers.classPrivateFieldSet2(_A, this, value);
 }], [dec, 1, "b", function () {
-  return babelHelpers.classPrivateFieldGet2(this, _B);
+  return babelHelpers.classPrivateFieldGet2(_B, this);
 }, function (value) {
-  babelHelpers.classPrivateFieldSet2(this, _B, value);
+  babelHelpers.classPrivateFieldSet2(_B, this, value);
 }]], []);
```

Comment:
I want to try to optimize them to avoid an extra method.
It's a little difficult to be honest, but worth trying. :)
```ts
[_init_a, _get_a, _set_a, _init_b, _get_b, _set_b, _initProto] = babelHelpers.applyDecs(_Foo, [[dec, 1, "a", 
bind(babelHelpers.classPrivateFieldGet2, _A), 
bind(babelHelpers.classPrivateFieldSet2, _A)], [dec, 1, "b", 
bind(babelHelpers.classPrivateFieldGet2, _B), 
bind(babelHelpers.classPrivateFieldSet2, _B)]], []);
```